tests/client/test_async_client.py

```diff
@@ -14,6 +14,18 @@
     assert repr(response) == "<Response [200 OK]>"
 
 
+async def test_build_request(server, backend):
+    url = server.url
+    async with httpx.AsyncClient(backend=backend) as client:
+        request = client.build_request("GET", url)
+        response = await client.send(request)
```

Comment:
Should probably modify the request in some way before sending it and then verify the request is modified in the result.

Reply:
do you have any suggestion for testing?

Reply:
Maybe adding a header, just something that wouldn't be done by default.